docs/src/lib/types.md

```diff
@@ -16,7 +16,12 @@ It is not intended as a fully generic interface for working with tabular data, w
 interfaces defined by [Tables.jl](https://github.com/JuliaData/Tables.jl/) instead.
 
 `DataFrame` is the most fundamental subtype of `AbstractDataFrame`, which stores a set of columns
-as `AbstractVector` objects.
+as `AbstractVector` objects. When a `DataFrame` is constructed columns *are not* copied, if possible.
```

Comment:
This should probably go to the new section too. We don't mention constructors for other types.

Reply:
OK - moved